### PR TITLE
set up repo

### PR DESCRIPTION
**Pull request by juliasloan25**
## Purpose 
Generic setup for the new repo, including folder structure and Project.toml files.

Closes #2

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

**Review by LenkaNovak — APPROVED** [review]
Hi @juliasloan25 , this LGTM. I'm not sure we'll need the experiments directory. What were you thinking would live here?

**Comment by juliasloan25** [comment]
> Hi @juliasloan25 , this LGTM. I'm not sure we'll need the experiments directory. What were you thinking would live here?

I'm not sure, maybe some examples showing how to use the codebase. But the tests should be enough to show the usage of all of the functions. I'll delete the experiments folder for now, and we can always add it back if we need it later.

File: .gitignore [hunk 1]
```diff
@@ -21,4 +21,6 @@ docs/site/
 # It records a fixed state of all packages used by the project. As such, it should not be
 # committed for packages, but should be committed for applications that require a static
 # environment.
-Manifest.toml
+/Manifest.toml
+.dev/Manifest.toml
```
Comment (charleskawczynski) [inline .gitignore:25]: This one might be nice to have?

File: test/runtests.jl [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (charleskawczynski) [inline test/runtests.jl:1]: Can we add a `@testset` (or better yet, start off with loading SafeTestsets + aqua tests)

**Pull request merged**